tests/e2e/gc_block_volume_expansion.go

```diff
@@ -1828,6 +2127,47 @@ func onlineVolumeResizeCheck(f *framework.Framework, client clientset.Interface,
 	// first time
 	gomega.Expect(fsSize).Should(gomega.BeNumerically(">", originalSizeInMb), fmt.Sprintf("error updating filesystem size for %q. Resulting filesystem size is %d", pvclaim.Name, fsSize))
 	ginkgo.By("File system resize finished successfully")
+
 	framework.Logf("Online volume expansion in GC PVC is successful")
 
 }
+
+func createStaticPVandPVCinGuestCluster(client clientset.Interface, ctx context.Context, namespace string, svpvcName string, size string, storageclass *storagev1.StorageClass) (*v1.PersistentVolumeClaim, *v1.PersistentVolume, *v1.Pod, string) {
+	ginkgo.By("Creating PV in guest cluster")
+	gcPV := getPersistentVolumeSpecWithStorageclass(svpvcName, v1.PersistentVolumeReclaimDelete, storageclass.Name, nil, size)
+	gcPV, err := client.CoreV1().PersistentVolumes().Create(ctx, gcPV, metav1.CreateOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	gcPVName := gcPV.GetName()
+	time.Sleep(time.Duration(10) * time.Second)
+	framework.Logf("PV name in GC : %s", gcPVName)
+
+	ginkgo.By("Creating PVC in guest cluster")
+	gcPVC := getPVCSpecWithPVandStorageClass(svpvcName, namespace, nil, gcPVName, storageclass.Name, size)
+	gcPVC, err = client.CoreV1().PersistentVolumeClaims(namespace).Create(ctx, gcPVC, metav1.CreateOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	time.Sleep(time.Duration(10) * time.Second)
```

Comment:
I do see there is a wait for PVC bind next, what are we waiting for here

Reply:
Deleted this